tests/integration_tests/modules/test_hotplug.py

```diff
@@ -308,17 +310,36 @@ def test_multi_nic_hotplug(client: IntegrationInstance):
 
 
 @pytest.mark.skipif(PLATFORM != "ec2", reason="test is ec2 specific")
-@pytest.mark.skip(reason="IMDS race, see GH-5373. Unskip when fixed.")
-def test_multi_nic_hotplug_vpc(session_cloud: IntegrationCloud):
+@pytest.mark.parametrize("test_early_plug", [False, True])
+def test_multi_nic_hotplug_vpc(
+    session_cloud: IntegrationCloud, test_early_plug: bool
+):
     """Tests that additional secondary NICs are routable from local
     networks after the hotplug hook is executed when network updates
     are configured on the HOTPLUG event."""
+    if test_early_plug:
+        # This test usually passes without wait_for_cloud_init(),
+        # but sometimes the hotplug event races with the end of cloud-init
+        # so occasionally fails. For now, skip it. Keep it around, because
+        # this variation of the test would be valuable to test early hotplug,
+        # if it is ever supported.
+        pytest.skip(
```

Comment:
I generally don't like adding skips in the middle of integration tests like this because we we wind up spending the time launching an instance to do nothing. How often do we actually inspect our skips and say "oh hey, we should go fix that?" Any more often than we inspect TODOs in the code? I would prefer we make this a TODO and not launch anything until we actually have behavior to test.